docs/en/standalone.md

```diff
@@ -0,0 +1,134 @@
+# Standalone Mode Guideline
+
+## Deployment
+### Modify Config
+1. Replace 127.0.0.1 with the real IP if may be accessed in another node
+2. If the specific port is unavailable, change to another port
+
+* conf/tablet.flags
+   ```
+   --endpoint=127.0.0.1:9921
+   ```
+* conf/nameserver.flags
+   ```
+   --endpoint=127.0.0.1:6527
+   # set the same value as the endpoint in conf/tablet.flags
+   --tablet=127.0.0.1:9921
+   #--zk_cluster=127.0.0.1:7181
+   #--zk_root_path=/openmldb_cluste
+   ```
+* conf/apiserver.flags
+   ```
+   --endpoint=127.0.0.1:8080
+   # set the same value as the endpoint in conf/nameserver.flags
+   --nameserver=127.0.0.1:6527
+   ```
+### Start service
+```bash
+sh bin/start-all.sh
+```
+If want to stop, execute the command as follows 
+```bash
+sh bin/stop-all.sh
+```
+## Usage
+### Start CLI
+```bash
+# The value of host and port is the endpoint in conf/nameserver.flags
+./openmldb --host 127.0.0.1 --port 6527
+```
+### Create DB
+```sql
+> CREATE DATABASE demo_db;
+```
+
+### Create Table
+```sql
+> USE demo_db;
+> CREATE TABLE demo_table1(c1 string, c3 int, c4 bigint, c5 float, c6 double, c7 timestamp, c8 date, index(ts=c7));
+```
+**Note**: Specify at least one index and set the ts column which is used for orderby
```

Comment:
orderby => ORDER BY

Reply:
`ts` is the key in `index(ts=c7)` option

Reply:
i see